src/datastore/memory-store.ts

```diff
@@ -1,8 +1,12 @@
-import { DataStore, DbBlock, DbTx } from './common';
+import { DataStore, DbBlock, DbTx, DbStxEvent, DbFtEvent, DbNftEvent, DbSmartContractEventTypeId } from './common';
 
 export class MemoryDataStore implements DataStore {
   readonly blocks: Map<string, DbBlock> = new Map();
   readonly txs: Map<string, DbTx> = new Map();
+  readonly stxEvents: Map<string, DbStxEvent> = new Map();
+  readonly ftEvents: Map<string, DbFtEvent> = new Map();
```

Comment:
I'd err on the side of being explicit here and not using acronym for readabilities sake